test/grub_install_test.rb

```diff
@@ -12,15 +12,7 @@ def stub_arch(arch)
     end
 
     def stub_efivars(removable: false)
-      efivardirs = if removable
-        []
-      else
-        ["Boot0000-8be4df61-93ca-11d2-aa0d-00e098032b8c",
-         "BootCurrent-8be4df61-93ca-11d2-aa0d-00e098032b8c",
-         "BootOrder-8be4df61-93ca-11d2-aa0d-00e098032b8c"]
-      end
-
-      allow(Dir).to receive(:glob).and_return(efivardirs)
+      allow(Bootloader::Systeminfo).to receive(:writable_efivars?).and_return(!removable)
```

Comment:
:thinking: So, if they are _removable_ they are _not writable_?

Reply:
well, it is kind of other way around. grub2-install has `--removable` flag that means do not write to efibootmanager. So basically do not modify efivars. This way we avoid problem with non writable efivars

Reply:
Ok, thanks for the explanation :)